### DISCUSSION TITLE
Print Paper in JointJs

### DISCUSSION THREAD

**Discussion by slvamegestic** [clientIO/joint#2000] [Q&A] [answered]
Hi all, 
 I am using JointJs 3.6.5.
 
I would like to print a paper using JointJS. In fact, I am able to download the Paper in SVG format by referring to link [#1502]

Is there is any way to do print the paper as same as print screen functionality? it would be great if its possible in jointJs.

**Comment by kumilingus** [comment, marked as answer]
The [print plugin](https://resources.jointjs.com/docs/rappid/v3.6/format.Print.html) is part of our commercial extension *JointJS+*.

Note, you can resize the paper to cover full window and then open print dialog with:
```
window.print();
```

**Reply by slvamegestic** [reply]
thanks, I'll look into it
